crates/gpui/src/platform/windows/platform.rs

```diff
@@ -203,6 +222,7 @@ impl Platform for WindowsPlatform {
         if let Some(ref mut callback) = self.state.borrow_mut().callbacks.quit {
             callback();
         }
+        unsafe { OleUninitialize() };
```

Comment:
`OleUninitialize()` hasn't been removed; it has been moved here. The reason for moving it is that running `OleUninitialize()` in `Drop` causes a `STATUS_ACCESS_VIOLATION` error during testing.

Reply:
It's been fixed.